packages/core/namespace.ts

```diff
@@ -33,7 +33,8 @@ type CubeCreatorClass =
   'UnlistJob' |
   'ReplaceCSVAction' |
   'MediaLocal' |
-  'MediaURL'
+  'MediaURL' |
+  'OriginalValuePredicate'
```

Comment:
Should be `OriginalValue` ?

Reply:
It's a type that we assign on the predicate pointing to the original value, so that's why I named it that.